dist/trip.js

```diff
@@ -1056,6 +1056,26 @@
         'e s w n screen-ne screen-se screen-sw screen-nw screen-center');
       $tripBlock.addClass(o.position);
 
+      // if we have a nextClickSelector use that as the trigger for the next button
+      if(o.nextClickSelector){
+        var eventName = (o.nextClickEvent || 'click')+'.Trip'
+        $(o.nextClickSelector).one(eventName, function(e) {
+          e.preventDefault();
+          // Force IE/FF to lose focus
+          $(this).blur();
+          that.next();
+        });
+
+        $tripBlock
+          .find('.trip-next')
+          .hide()
+
+      }else{
+        $tripBlock
+          .find('.trip-next')
+          .show()
+      }
```

Comment:
one few lines below, you will notice that we will decide whether to show / hide ".trip-next" by some rules. Can you move trip-next related parts to there ?